### PR TITLE
chore: Build Ubuntu 18 binaries

### PR DESCRIPTION
**Pull request by pcnc**
* Lots of Ubuntu 18-specific handling
* Also fixes handling for the `ansible_arguments` Packer var since that never actually worked

### CODE REVIEW

File: migrations/schema.sql [hunk 1]
```diff
@@ -135,6 +142,20 @@ CREATE EXTENSION IF NOT EXISTS pgjwt WITH SCHEMA extensions;
 COMMENT ON EXTENSION pgjwt IS 'JSON Web Token API for Postgresql';
 
 
+--
```
Comment (darora) [inline migrations/schema.sql]: schema changes seem unrelated to u18 concerns?

**Review by darora — APPROVED** [review]

File: migrations/schema.sql (hunk 1 above)
Comment (pcnc) [reply migrations/schema.sql]: Yup, those were likely generated by a previous merge commit.
Fixed by merging `develop` into this branch
